packages/playground/src/components/user-management/form-user-delete-dialog.tsx

```diff
@@ -40,9 +28,9 @@ export const FormDeleteUserDialog: React.FC<FormResetPasswordrDialogProps> = ({
       <AlertDialogTrigger asChild></AlertDialogTrigger>
       <AlertDialogContent>
         <AlertDialogHeader>
-          <AlertDialogTitle>Are you absolutely sure to remove {user.display_name}?</AlertDialogTitle>
+          <AlertDialogTitle>Are you absolutely sure to remove {user?.display_name}?</AlertDialogTitle>
           <AlertDialogDescription>
-            This will permanently remove {user.display_name} in the system.
+            This will permanently remove {user?.display_name} in the system.
```

Comment:
`This will permanently remove  the user "{user?.display_name}" from the system.`

Reply:
fixed

Reply:
Just change "" to a `badge` component in the title for users to focus on the user name easier. How do you think this idea? 
cc: @Abhinav Rastogi 
<img width="760" alt="Screenshot 2024-10-25 at 3 55 20 PM" src="https://github.com/user-attachments/assets/c0172699-20ff-4ae4-b6c5-a92db83edab1">
<img width="763" alt="Screenshot 2024-10-25 at 3 55 28 PM" src="https://github.com/user-attachments/assets/61ebf054-093f-4ccd-ae2f-3cac52974248">